content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -102,13 +102,22 @@ to ensure that the API presents a clear, consistent view of system resources
 and behavior, and to enable controlling access to end-of-life and/or
 experimental APIs.
 
-Refer to [API versions reference](/docs/reference/using-api/api-overview/#api-versioning)
-for more details on the API version level definitions.
-
 To make it easier to evolve and to extend its API, Kubernetes implements
 [API groups](/docs/reference/using-api/api-overview/#api-groups) that can be
 [enabled or disabled](/docs/reference/using-api/api-overview/#enabling-or-disabling).
 
+API resources are distinguished by their API group, resource type, namespace
+(for namespaced resources), and name. The API server may serve the same
```

Comment:
I have remaining questions about how some of this content may be interpreted, but think
these changes could move forward.

What is exactly is a resource "type"? The provided example did not fully clear up my questions about the relationship between `kind` and type. It seems important to clearly document how the types are configured and defined in requests/responses.

There are a few style nits:
- no latin phrases (via)
- try not to use the word, "simply"
- use plural

The API server may serve the same underlying data through different API versions
and handle the conversion between the API versions transparently. The different versions of an
API are capable of representing the same resource. For example, ...

/approve